include/yaclib/fault/detail/antagonist/yielder.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+// TODO(myannayx): define in cmake depending on system
+#define kSleepTime 500
+
+#include <yaclib/fault/thread.hpp>
+
+#include <atomic>
+#include <random>
+
+namespace yaclib::detail {
+// TODO(myannyax) stats?
+class Yielder {
+ public:
+  explicit Yielder(int frequency);
+  void MaybeYield();
+
+ private:
+  bool ShouldYield();
+  void Reset();
+  unsigned RandNumber(unsigned max);
```

Comment:
Why unsigned?

Reply:
because it's used with unsigned numbers here
though I forgot to change atomic_int to atomic_uint